### ISSUE TITLE
Remember credentials

### ISSUE THREAD

**Issue by mxm199** [savvasdalkitsis/uhuruphotos-android#630]
Good day to all
Although in the latest versions of the application, the function to Remember credentials has appeared, but still the credentials are not saved, saving may work for one or two days, but then it still asks you to enter the data, and from the initial window for selecting the use of photos on the device or personal cloud. It's very inconvenient.

**Comment by Sprintrio** [comment]
Same issue here; but just from the last update

**Comment by td94** [comment]
Same problem. I had a short look at the code and it looks like the credentials should be stored. Looking at the db I cannot find anything related to credentials inside.

Sidenote:
It looks like (even when credentials would be stored correctly) they are deleted on connection loss and one retry. Maybe it would make sense to keep them stored, but just stop retries (as this often will be related to libre photos located in a specific network and not always available). Another helpful functionality could be to sync only in a specific WiFi SSID.

But: I downloaded the app yesterday and I'm generally quite impressed! Great work so far!

**Comment by nagarrido** [comment]
Same problem here. (I can also say, as the last comment, that I'm loving the latest changes of the app)

**Comment by judgedread540** [comment]
I'm also having issues. I get logged out every few hours and the credentials are never saved.

**Comment by mxm199** [comment]
Yes, I noticed that if you log out of the web interface, or when the authorization tokens are "rotten", then it is impossible to log in through the application (until you log in through the site), or it resets authorization

**Comment by nagarrido** [comment]
Any progress on this? I have to log in every time I use the app 😔

**Comment by CodingGhost** [comment]
same issue... cant use it in this state..

**Comment by MicheleM42** [comment]
Hi, same here, could you please fix the issue? Thanks

**Comment by savvasdalkitsis** [comment]
This seems to be caused by minifying. I will be trying to figure this out. In the meantime i will release a non minified version to fix this for people but I will not close this issue until i fix it in a minified app. Thanks to John Warren from the librephotos discord for nailing this down to minifying

**Comment by nagarrido** [comment]
Worked for me!
